src/openrct2/input.c

```diff
@@ -90,7 +90,7 @@ static void game_handle_input_mouse(sint32 x, sint32 y, sint32 state);
 static void input_widget_left(sint32 x, sint32 y, rct_window *w, rct_widgetindex widgetIndex);
 void input_state_widget_pressed(sint32 x, sint32 y, sint32 state, rct_widgetindex widgetIndex, rct_window* w, rct_widget* widget);
 void set_cursor(uint8 cursor_id);
-static void input_window_position_continue(rct_window *w, sint32 lastX, sint32 lastY, sint32 newX, sint32 newY);
+static void input_window_position_continue(rct_window *w, sint32 wdx, sint32 wdy, sint32 x, sint32 y);
```

Comment:
wdx? What is that supposed to mean?

Reply:
You're right. I changed it. My IDE just tells me the names of the parameters are different in the implementation and declaration.